<launch>
    <arg name="load_params_from_file" default="true" />
    <arg name="param_file" default="$(find strands_ground_hog)/config/ground_hog.yaml" />
    <rosparam command="load" file="$(arg param_file)" if="$(arg load_params_from_file)"/>
    
    <arg name="camera_namespace" default="/head_xtion" />
    <arg name="detections" default="/groundHOG/detections" />
    <arg name="result_image" default="/groundHOG/image" />
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>

    <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH)"/>

    <node pkg="strands_ground_hog" type="groundHOG" name="ground_hog" output="log">
        <param name="ground_plane" value="" type="string"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="detections" value="$(arg detections)" type="string"/>
        <param name="result_image" value="$(arg result_image)" type="string"/>
    </node>

</launch> 
